tt_metal/hw/ckernels/wormhole_b0/common/src/ckernel.cc

```diff
@@ -0,0 +1,221 @@
+
```

Comment:
this file doesn't exist in the metal repo anymore and won't get compiled, shouldn't get recreated.